wix/Main.wxs.template

```diff
@@ -178,51 +178,51 @@
           Key="SOFTWARE\JavaSoft\$(var.OracleJavasoftBaseKey)\$(var.OracleVersionKey)"
           Name="JavaHome" Type="string" Value="[INSTALLDIR]" KeyPath="yes" />
         <?if $(env.PRODUCT_CATEGORY)="jre"?>
-        <RegistryValue Root="HKLM"
-          Key="SOFTWARE\JavaSoft\$(var.OracleJavasoftBaseKey)\$(var.OracleVersionKey)"
-          Name="RuntimeLib" Type="string" Value="[INSTALLDIR]$(var.DllPath)" KeyPath="no" />
-        <!-- Oracle dont set RuntimeLib for JDK .. because JRE is "private" JRE so no ?else? -->
+          <RegistryValue Root="HKLM"
+            Key="SOFTWARE\JavaSoft\$(var.OracleJavasoftBaseKey)\$(var.OracleVersionKey)"
+            Name="RuntimeLib" Type="string" Value="[INSTALLDIR]$(var.DllPath)" KeyPath="no" />
+          <!-- Oracle doesn't set RuntimeLib for JDK .. because JRE is "private" JRE so no ?else? -->
         <?endif?>
         <RegistryValue Root="HKLM"

```

Comment:
Whitespace for this doesn't match your change above?

Reply:
@karianna I am unsure which section you are referring to about the whitespace mismatch. I will say, some of the whitespace tweaks were done by @gdams, but I wanted to change back the inner blocks of if-statements to be indented so the reader is more clear on which lines fall within the if-statement block

Reply:
@jmjaffe37 feel free to tweak them to however you prefer